plugins/communications-app/RecipientsForm/index.jsx

```diff
@@ -0,0 +1,187 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Form } from '@openedx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { useSelector, useDispatch } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context';
+import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
+
+import './styles.scss';
+
+const disableIsHasLearners = ['track', 'cohort'];
+const recipientsFormDescription = 'A selectable choice from a list of potential email recipients';
+
+const RecipientsForm = ({ cohorts: additionalCohorts, courseModes }) => {
+  const formData = useSelector((state) => state.form);
+  const dispatch = useDispatch();
+  const { isEditMode, emailRecipients, isFormSubmitted } = formData;
+  const hasCourseModes = courseModes.length > 1;
```

Comment:
In the current implementation this has null checking

https://github.com/openedx/frontend-app-communications/blob/6dd835d63f106c6e1bf7a02de16d3047a5bee558/src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/BulkEmailRecipient.jsx#L23

Is there a specific reason that was removed?

Reply:
The reason is because `courseModes` are optional as prop and by default has an empty array so is not necessary to make the null validation